### PR TITLE
fix annoying PRESEQ failure for methylseq

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/preseq/lcextrap/main.nf [hunk 1]
```diff
@@ -21,6 +21,7 @@ process PRESEQ_LCEXTRAP {
 
     script:
     def args = task.ext.args ?: ''
+    args = task.attempt > 1 ? args.join(' -defects') : args
```
Comment (ewels) [inline modules/nf-core/preseq/lcextrap/main.nf:24]: https://preseq.readthedocs.io/en/latest/#lc_extrap

```suggestion
    args = task.attempt > 1 ? args.join(' -defects') : args  // Disable testing for defects
```

**Review by ewels — APPROVED** [review]
Thank you!

**Comment by edmundmiller** [comment]
Shouldn't this be upstreamed to modules or this should have bumped the version?

**Comment by ewels** [comment]
> Shouldn't this be upstreamed to modules

This _is_ modules...? I don't understand.

I don't think that a version bump is required to add this flag.

**Comment by ewels** [comment]
> I don't think that a version bump is required to add this flag.

Though it looks like 3.2.0 is available..

**Comment by edmundmiller** [comment]
> This is modules...? I don't understand.

Don't mind me, saw methylseq in the title and misread it I guess 😅

**Comment by sateeshperi** [comment]
@maxulysse / @ewels why was the strategy changed from ignore to retry here ?

**Comment by edmundmiller** [comment]
Looks like it's still open https://github.com/nf-core/methylseq/issues/161

**Comment by sateeshperi** [comment]
yes so, ignore makes sense....not sure why it was changed to retry in this PR

**Comment by maxulysse** [comment]
I think maybe ignore was too strict. I honestly don't remember...

**Comment by sateeshperi** [comment]
I think maybe because the intent was to retry with the defect mode with this line
`args = task.attempt > 1 ? args.join(' -defects') : args  // Disable testing for defects`

**Pull request merged**